### PR TITLE
E2E: Add more test cases for Webring

### PR DESCRIPTION
**Pull request by emperol2**
Hi @dtinth ,

I would like to submit E2E and add more test cases for Webring.

- e2e with real user behavior on a random link
- should navigate to the next link for n-link or last-link
- should navigate to the previous link for n-link or first-link
- should display the sticky consent banner
- should hide the sticky consent banner when it is clicked
- should contain a link on-site info section
- should return 200 for data.json
- should have keys for data.json

Feel free to change :smile:

Thanks

### CODE REVIEW

File: automation/e2e/cypress/integration/webring_e2e.spec.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
+/// <reference types="cypress" />
+
+function commonTests() {
+  context("e2e with real user behaviour on random link", () => {
+    let baseURL = "https://webring.wonderful.software/"
```
Comment (dtinth) [inline automation/e2e/cypress/integration/webring_e2e.spec.js:5]: The baseUrl from `cypress.json` is preferred.

File: automation/e2e/cypress/integration/webring_e2e.spec.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+})
+
+context("Desktop", () => {
+  beforeEach(() => {
+    cy.viewport(1024, 768)
+  })
+  commonTests()
+})
+
+context("data.json", () => {
```
Comment (dtinth) [inline automation/e2e/cypress/integration/webring_e2e.spec.js:100]: Maybe this part can be its own file?

**Review by dtinth — APPROVED** [review]
@emperol2 Thanks for your contribution, this covers a lot of cases!

**Comment by emperol2** [comment]
@dtinth fixed, please merge when you are ready 👍

**Review by dtinth — APPROVED** [review]
Thank you!!

**Pull request merged**